src/proxies/ClientProxy.ts

```diff
@@ -14,33 +11,12 @@ export default class ClientProxy {
             url: `/reg/${clientId}?access_token=${client.registrationAccessToken}`,
         });
 
-        if (r.status !== 200) {
-            throw new ClientProxyError(r.data);
-        }
-
         client.clientSecret = r.data['client_secret'];
         client.requestUris = r.data['request_uris'];
 
         return client;
     }
 
-    static async info(clientId: string, registrationAccessToken: string) {
```

Comment:
@GarfDev This service was kind of a duplicate for the `get()` method. I have removed it and changed the controller implementation as you can read in my previous comments. Stay DRY!😄